If we are not using setup.cfg, remove it

1. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1b5c49c3904dffe63546adbadc47759a53fbc8ca/setup.cfg#L1-L2
  
    This is applied when [creating wheel files](https://stackoverflow.com/q/31573107) for a Python package. It denotes a universal wheel, i.e. one that is platform independent. 
    
    SCT does not create wheels for its `spinalcordtoolbox` package, so this is unnecessary and can be removed.

2. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1b5c49c3904dffe63546adbadc47759a53fbc8ca/setup.cfg#L4-L5

    This setting only exists in the context of a _very_ outdated copy of Pytest's documentation: https://docs.pytest.org/en/4.6.x/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner
    
    We do not use `pytest-runner`, though, nor do we ever execute tests via `python setup.py test`. So, this can be removed.

3. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1b5c49c3904dffe63546adbadc47759a53fbc8ca/setup.cfg#L19-L23
    
    [`isort`](https://pycqa.github.io/isort/docs/configuration/config_files.html)  is a code-formatting library, similar to `black`, that imposes a strict style on imports at the top of scripts. We don't currently install or use `isort`. We could, but we don't.

4. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/99f7f9413a7ce2536273b85611aaf8601a96c8da/setup.cfg#L18-L19

    We don't use `.tox`. But, more importantly, we don't even run `flake8` on the root directory; we only run it on git-tracked files:

    https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1b5c49c3904dffe63546adbadc47759a53fbc8ca/.github/workflows/lint_code.yml#L22

    So, the `exclude` setting can be removed.

6. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1b5c49c3904dffe63546adbadc47759a53fbc8ca/setup.cfg#L15-L16

    * [`E501`](https://www.flake8rules.com/rules/E501.html): Line too long (82 > 79 characters)
    * [`E402`](https://www.flake8rules.com/rules/E402.html): Module level import not at top of file
    
    E501 makes sense, given we use nonstandard line lengths in our Python files. 

    E402, however was added in [#1311](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/1311), but currently, it suppresses only a few warnings, all in `registration/algorithms.py`, due to [VoxelMorph's backend settings](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1b5c49c3904dffe63546adbadc47759a53fbc8ca/spinalcordtoolbox/registration/algorithms.py#L24-L27). I think we could easily convert this E402 global exclusion into a `# noqa` comment for VoxelMorph only.

7. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/99f7f9413a7ce2536273b85611aaf8601a96c8da/setup.cfg#L21

    This is meaningless given that we already specify `E501` in the ignore list. 

    If we were to remove `E501`, however, then `flake8` would throw errors for line lengths `>179`. There are a lot of these, apparently!

    <details>
    ```console
    (venv_sct) (master) joshua@tadpole:~/repos/spinalcordtoolbox$ flake8 spinalcordtoolbox | grep E501
    spinalcordtoolbox/deepseg/models.py:104:180: E501 line too long (202 > 179 characters)
    spinalcordtoolbox/image.py:824:180: E501 line too long (284 > 179 characters)
    spinalcordtoolbox/labels.py:25:180: E501 line too long (187 > 179 characters)
    spinalcordtoolbox/math.py:329:180: E501 line too long (185 > 179 characters)
    spinalcordtoolbox/registration/algorithms.py:885:180: E501 line too long (209 > 179 characters)
    spinalcordtoolbox/registration/algorithms.py:896:180: E501 line too long (207 > 179 characters)
    spinalcordtoolbox/registration/algorithms.py:1278:180: E501 line too long (195 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:86:180: E501 line too long (196 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:187:180: E501 line too long (220 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:188:180: E501 line too long (220 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:228:180: E501 line too long (229 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:229:180: E501 line too long (227 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:230:180: E501 line too long (186 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:242:180: E501 line too long (220 > 179 characters)
    spinalcordtoolbox/registration/landmarks.py:243:180: E501 line too long (220 > 179 characters)
    spinalcordtoolbox/scripts/sct_analyze_lesion.py:241:180: E501 line too long (212 > 179 characters)
    spinalcordtoolbox/scripts/sct_analyze_lesion.py:377:180: E501 line too long (238 > 179 characters)
    spinalcordtoolbox/scripts/sct_analyze_texture.py:296:180: E501 line too long (209 > 179 characters)
    spinalcordtoolbox/scripts/sct_analyze_texture.py:297:180: E501 line too long (196 > 179 characters)
    spinalcordtoolbox/scripts/sct_compute_hausdorff_distance.py:265:180: E501 line too long (191 > 179 characters)
    spinalcordtoolbox/scripts/sct_dmri_moco.py:26:180: E501 line too long (239 > 179 characters)
    spinalcordtoolbox/scripts/sct_image.py:150:180: E501 line too long (218 > 179 characters)
    spinalcordtoolbox/scripts/sct_image.py:162:180: E501 line too long (218 > 179 characters)
    spinalcordtoolbox/scripts/sct_label_utils.py:14:180: E501 line too long (187 > 179 characters)
    spinalcordtoolbox/scripts/sct_maths.py:495:180: E501 line too long (191 > 179 characters)
    spinalcordtoolbox/scripts/sct_register_to_template.py:118:180: E501 line too long (391 > 179 characters)
    spinalcordtoolbox/scripts/sct_register_to_template.py:629:180: E501 line too long (188 > 179 characters)
    spinalcordtoolbox/scripts/sct_register_to_template.py:734:180: E501 line too long (197 > 179 characters)
    spinalcordtoolbox/scripts/sct_smooth_spinalcord.py:207:180: E501 line too long (247 > 179 characters)
    spinalcordtoolbox/scripts/sct_smooth_spinalcord.py:216:180: E501 line too long (196 > 179 characters)
    spinalcordtoolbox/scripts/sct_smooth_spinalcord.py:240:180: E501 line too long (192 > 179 characters)
    spinalcordtoolbox/scripts/sct_straighten_spinalcord.py:143:180: E501 line too long (267 > 179 characters)
    spinalcordtoolbox/scripts/sct_straighten_spinalcord.py:144:180: E501 line too long (212 > 179 characters)
    spinalcordtoolbox/scripts/sct_warp_template.py:40:180: E501 line too long (189 > 179 characters)
    spinalcordtoolbox/types.py:87:180: E501 line too long (191 > 179 characters)
    spinalcordtoolbox/types.py:216:180: E501 line too long (1275 > 179 characters)
    spinalcordtoolbox/types.py:567:180: E501 line too long (229 > 179 characters)
    spinalcordtoolbox/types.py:569:180: E501 line too long (210 > 179 characters)
    spinalcordtoolbox/types.py:573:180: E501 line too long (230 > 179 characters)
    ```
    </details>

    So, I think I would actually suggest removing `E501` from our exclude list, then begin policing line lengths.

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,3 @@
-[bdist_wheel]
-universal = 1
-
 [aliases]
 test=pytest
 

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,3 @@
-[aliases]
-test=pytest
-
 [tool:pytest]
 addopts = --verbose --show-capture=stderr --tb=native --disable-warnings
 python_files = testing/**/test_*.py

--- a/setup.cfg
+++ b/setup.cfg
@@ -9,9 +9,3 @@ exclude = build,.git,.tox,
 # extend-ignore preserves flake8's default ignore list: W503,W504,E24,E121,E123,E126,E226,E704
 extend-ignore = E501,E402
 max-line-length = 179
-
-[isort]
-combine_as_imports = true
-default_section = THIRDPARTY
-include_trailing_comma = true
-multi_line_output = 5

--- a/setup.cfg
+++ b/setup.cfg
@@ -5,6 +5,4 @@ markers =
     sct_testing: marks tests that belonged to previous testing harness (often used by users to verify installation)
 
 [flake8]
-# extend-ignore preserves flake8's default ignore list: W503,W504,E24,E121,E123,E126,E226,E704
-extend-ignore = E501
 max-line-length = 179

--- a/setup.cfg
+++ b/setup.cfg
@@ -5,7 +5,6 @@ markers =
     sct_testing: marks tests that belonged to previous testing harness (often used by users to verify installation)
 
 [flake8]
-exclude = build,.git,.tox,
 # extend-ignore preserves flake8's default ignore list: W503,W504,E24,E121,E123,E126,E226,E704
 extend-ignore = E501,E402
 max-line-length = 179

--- a/setup.cfg
+++ b/setup.cfg
@@ -6,5 +6,5 @@ markers =
 
 [flake8]
 # extend-ignore preserves flake8's default ignore list: W503,W504,E24,E121,E123,E126,E226,E704
-extend-ignore = E501,E402
+extend-ignore = E501
 max-line-length = 179

--- a/spinalcordtoolbox/registration/algorithms.py
+++ b/spinalcordtoolbox/registration/algorithms.py
@@ -21,10 +21,6 @@ from scipy.signal import argrelmax, medfilt
 from sklearn.decomposition import PCA
 from scipy.io import loadmat
 import torch
-# import VoxelMorph and Neurite (used in VoxelMorph) with pytorch backend
-os.environ['VXM_BACKEND'] = 'pytorch'
-os.environ['NEURITE_BACKEND'] = 'pytorch'
-import voxelmorph as vxm
 
 import spinalcordtoolbox.image as image
 from spinalcordtoolbox.math import laplacian
@@ -34,6 +30,11 @@ from spinalcordtoolbox.scripts import sct_resample
 from spinalcordtoolbox.utils import sct_progress_bar, copy_helper, run_proc, tmp_create, sct_dir_local_path
 
 from spinalcordtoolbox.scripts import sct_image
+
+# import VoxelMorph and Neurite (used in VoxelMorph) with pytorch backend
+os.environ['VXM_BACKEND'] = 'pytorch'
+os.environ['NEURITE_BACKEND'] = 'pytorch'
+import voxelmorph as vxm  # noqa: E402
 
 # TODO [AJ]
 # introduce potential cleanup functions in case exceptions occur and

